Makefile

```diff
@@ -1,5 +1,6 @@
 PYTHON2_CMD   ?= python2
 PYTEST         = py.test
+BLACK          = black
```

Comment:
I think you either forgot to remove this line or forgot to use the variable below.